### PR TITLE
[Fixes #7052 and #7037] Improved extensions ethereum support

### LINKED ISSUE #7052: Improve data returned by log events
### User Story

As a developer, I want to easily navigate ethereum logs data

### Description

*Type*: Bug

### PR DESCRIPTION
**Pull request by jeluard**
fixes #7052 and #7037

### Summary:

Refactored `ethereum` events to introduce `on-sucess` and `on-failure` events. Fixed some low-hanging fruits.

status: ready

### CODE REVIEW

**Comment by ghost** [comment]
### Pull Request Checklist
- [x] Have you updated the documentation, if impacted (e.g. [docs.status.im](https://docs.status.im/docs/build_status.html))?

File: deps.edn [hunk 1]
```diff
@@ -11,7 +11,7 @@
          com.taoensso/timbre         {:mvn/version "4.10.0"}
          hickory                     {:mvn/version "0.7.1"}
          com.cognitect/transit-cljs  {:mvn/version "0.8.248"}
-         status-im/pluto             {:mvn/version "iteration-4-5"}
+         status-im/pluto             {:mvn/version "iteration-4-6-SNAPSHOT"}
```
Comment (flexsurfer) [inline deps.edn]: why SNAPSHOT" ?

File: src/status_im/extensions/core.cljs [hunk 2]
```diff
@@ -8,6 +8,7 @@
             [status-im.chat.commands.impl.transactions :as transactions]
             [status-im.ui.components.button.view :as button]
             [status-im.ui.components.checkbox.view :as checkbox]
+            [status-im.ui.components.icons.vector-icons :as icons]
```
Comment (flexsurfer) [inline src/status_im/extensions/core.cljs:11]: duplication

**Review by flexsurfer — APPROVED** [review]

File: deps.edn (hunk 1 above)
Comment (jeluard) [reply deps.edn]: Yes will change but waiting for some extra fixes. But good point will change in the PR

File: src/status_im/extensions/core.cljs (hunk 2 above)
Comment (jeluard) [reply src/status_im/extensions/core.cljs:11]: :+1:

**Review by goranjovic — APPROVED** [review]

**Comment by mandrigin** [comment]
@jeluard now all Jenkins jobs are failed. Why was it merged without passing CI?

**Comment by mandrigin** [comment]
@jeluard I'm reverting it, please fix tests and submit the PR again.

**Comment by mandrigin** [comment]
```
Testing status-im.test.extensions.ethereum

ERROR in (test-parse-topic-events) (TypeError:NaN:NaN)
topic parsing check - events
expected: (= ["0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef" "0x29f31b934498b0deabbe211b01cc30eee6475abf0f0d553e7eb8ca71deeb24b3"] (ethereum/parse-topic [{:event "Transfer", :params ["address" "address" "uint256"]} {:event "drawBet", :params ["uint256" "uint8" "int8" "address" "uint256"]}]))
  actual: #object[TypeError TypeError: Cannot read property 'call' of undefined]

ERROR in (test-parse-topic-indexed-values-address) (TypeError:NaN:NaN)
topic parsing check - indexed values address
expected: (= ["0x000000000000000000000000299b18709d4aacbda99048721448f65893a0c73a" "0x00000000000000000000000094eaa5fa6b313968b2abd6da375ef28077d95d53"] (ethereum/parse-topic {:type "address", :values ["0x299b18709d4aacbda99048721448f65893a0c73a" "0x94eaa5fa6b313968b2abd6da375ef28077d95d53"]}))
  actual: #object[TypeError TypeError: Cannot read property 'call' of undefined]

ERROR in (test-parse-topic-indexed-values-uint256) (TypeError:NaN:NaN)
topic parsing check - indexed values uint256
expected: (= ["0x00000000000000000000000000000000000000000000000000000000000003fd" "0x0000000000000000000000000000000000000000000000000000000000000078"] (ethereum/parse-topic {:type "uint256", :values ["0x3fd" "120"]}))
  actual: #object[TypeError TypeError: Cannot read property 'call' of undefined]

ERROR in (test-parse-topic-indexed-values-direct) (TypeError:NaN:NaN)
topic parsing check - indexed values direct
expected: (= ["0x00000000000000000000000000000000000000000000000000000000000003fd" "0x0000000000000000000000000000000000000000000000000000000000000078"] (ethereum/parse-topic ["0x00000000000000000000000000000000000000000000000000000000000003fd" "0x0000000000000000000000000000000000000000000000000000000000000078"]))
  actual: #object[TypeError TypeError: Cannot read property 'call' of undefined]

ERROR in (test-parse-topic-indexed-nil) (TypeError:NaN:NaN)
topic parsing check - nil
expected: (= ["0x00000000000000000000000000000000000000000000000000000000000003fd" nil] (ethereum/parse-topic ["0x00000000000000000000000000000000000000000000000000000000000003fd" nil]))
  actual: #object[TypeError TypeError: Cannot read property 'call' of undefined]
```

**Comment by mandrigin** [comment]
revert PR is here: https://github.com/status-im/status-react/pull/7163

**Pull request merged**